docs/auth.md

```diff
@@ -8,14 +8,14 @@ Manage authentication options.
 
 ## `apimetrics auth login`
 
-Login to APImetrics
+Login to APImetrics CLI.
```

Comment:
This would need to be changed in the source code of the auth login in the description property